pkg/transport/managed_transport.go

```diff
@@ -396,10 +326,19 @@ func (mt *ManagedTransport) getConn() network.Conn {
 	return conn
 }
 
+// updates underlying connection inactive deadline
+func (mt *ManagedTransport) updateConnDeadline() {
```

Comment:
I would tear the deadlines out as we discussed. We currently are not setting them anywhere and adding them back if necessary should be easy enough. So essentially this is dead code. As discussed in DMs, this is mostly necessary for `sudph` as it cant be closed properly atm so we should look into that specific transport first.